### PR TITLE
Finished Rick and Morty Exercise

### PR DESCRIPTION
**Pull request by soerendolling**

### CODE REVIEW

File: src/lib/character.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+/* eslint-disable quotes */
+/* eslint-disable semi */
+
+export function getCharacter() {
+  const filterMenu = document.querySelector(".header-filter__dropdown");
+  const characterStatus = filterMenu.value;
+  const url = createUrl(characterStatus);
+
+  console.log(url);
```
Comment (mpagels) [inline src/lib/character.js:9]: You can delete the `console.log` function
```suggestion
```

File: src/lib/character.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
+/* eslint-disable quotes */
+/* eslint-disable semi */
+
+export function getCharacter() {
+  const filterMenu = document.querySelector(".header-filter__dropdown");
+  const characterStatus = filterMenu.value;
+  const url = createUrl(characterStatus);
```
Comment (mpagels) [inline src/lib/character.js:7]: Very good to create a function which is returning an url

File: src/lib/character.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    console.log(url);
+  } else if (characterStatus === "unknown") {
+    url = `${baseUrl}?name=${searchInput}&status=unknown`;
+    console.log(url);
+  } else if (characterStatus === "all") {
+    url = `${baseUrl}?name=${searchInput}`;
+    console.log(url);
+  }
+  return url;
+}
```
Comment (mpagels) [inline src/lib/character.js:94]: The file is getting very long. I would recommend to create a new file and move the code in there and export the function and import it here back.

File: src/lib/character.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+export function clearCharacters() {
+  const characterElements = document.querySelectorAll(".main-character-card");
+  characterElements.forEach((article) => article.remove());
+}
```
Comment (mpagels) [inline src/lib/character.js:99]: I would create another file as well and export it from there

**Review by mpagels — APPROVED** [review]
Very good Sören. Found some tiny things. Most of them are refactor recommendations.
Amazing work!

File: src/lib/character.js (hunk 4 above)
Comment (soerendolling) [reply src/lib/character.js:99]: I will do, thanks for commenting

**Pull request closed without merging**